lib/syntax_tools/src/erl_syntax.erl

```diff
@@ -7726,7 +7820,8 @@ subtrees(T) ->
                     [[bitstring_type_m(T)],
                      [bitstring_type_n(T)]];
 		block_expr ->
-		    [block_expr_body(T)];
+		    [[block_expr_body(T)],
+		     block_expr_clauses(T)];
```

Comment:
this part here is going to cause a lot of issues because all block expressions going through the subtrees(T) call will now return 2 elements instead of 1. There may be a need to peek at the subtree to dynamically return the proper trype :/